subxt/src/extrinsic/params.rs

```diff
@@ -31,7 +32,7 @@ pub use sp_runtime::generic::Era;
 /// "additional" parameters that are signed and used in transactions.
 /// see [`BaseExtrinsicParams`] for an implementation that is compatible with
 /// a Polkadot node.
-pub trait ExtrinsicParams<T: Config> {
+pub trait ExtrinsicParams<T: Config>: Debug {
```

Comment:
Just wodnering why this is needed?

Reply:
See comment below :point_down: 

Chose to do on the trait itself rather than the method for a better error message.